python/ray/data/read_api.py

```diff
@@ -1146,73 +1332,72 @@ def read_tfrecords(
     ignore_missing_paths: bool = False,
     tf_schema: Optional["schema_pb2.Schema"] = None,
 ) -> Dataset:
-    """Create a dataset from TFRecord files that contain
+    """Create a :class:`~ray.data.Dataset` from TFRecord files that contain
     `tf.train.Example <https://www.tensorflow.org/api_docs/python/tf/train/Example>`_
     messages.
 
     .. warning::
-        This function exclusively supports ``tf.train.Example`` messages. If a file
+        This method exclusively supports ``tf.train.Example`` messages. If a file
         contains a message that isn't of type ``tf.train.Example``, then this function
-        errors.
+        will fail.
```

Comment:
```suggestion
       fails.
```
https://developers.google.com/style/tense